lsp/java/src/main/java/com/itsaky/lsp/java/actions/common/RemoveUnusedImportsAction.java

```diff
@@ -2,15 +2,20 @@
 
 import com.google.googlejavaformat.java.FormatterException;
 import com.google.googlejavaformat.java.RemoveUnusedImports;
+
 import com.itsaky.androidide.actions.ActionData;
 import com.itsaky.androidide.app.BaseApplication;
+import com.itsaky.androidide.utils.ILogger;
 import com.itsaky.lsp.java.actions.BaseCodeAction;
 import com.itsaky.lsp.java.R;
+
 import io.github.rosemoe.sora.text.Content;
 import io.github.rosemoe.sora.widget.CodeEditor;
 
 public class RemoveUnusedImportsAction extends BaseCodeAction {
```

Comment:
As written in contribution guidelines, files must be formatted with 4-space indentations. There is a simple Gradle task for that.